Routing/Loader/RestRouteLoader.php

```diff
@@ -117,6 +118,12 @@ private function getControllerLocator($controller)
         if ($this->container->has($controller)) {
             // service_id
             $prefix = $controller.':';
+
+             // sf > 4.1 support
+            if (method_exists(Kernel::class, 'getAnnotatedClassesToCompile')) {
```

Comment:
In the `Kernel` class we could more easily check the values of the version related constants. :)

Reply:
Indeed, see https://github.com/FriendsOfSymfony/FOSRestBundle/pull/1907 :)